oci/python/image.yaml

```
@@ -0,0 +1,11 @@
+version: 1
+
+upload:
+  - source: canonical/chiselled-python
+    commit: 3ce55f41e0e33ce1db3552fb18e8179c4a29941a
+    directory: python3.8/
+    release:
+      3.8.10-20.04:
```

```suggestion
      3.8-20.04:
```

I think you've removed the wrong one. It's less error prone to just merge the suggestions

---

Indeed, yeah, sorry about that. Updated.